release_notes.md

```diff
@@ -0,0 +1,38 @@
+ # Release notes
+
+- Woof Mobile APP
+
+All notable changes to this application will be documented in this file.
+
+## [Unreleased]
+
+`More information about working with the application will be added during development.`
```

Comment:
Why do we need this line here?
```suggestion
```

Reply:
We decided  with the team to to delete this information. In previous review you wrote that you also don't mind. Now the file release_notes.md keeps only the title of the file and the app name, because we haven't made any releases yet.